client/src/templates/Challenges/components/help-modal.tsx

```diff
@@ -1,16 +1,16 @@
 import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { Modal } from '@freecodecamp/react-bootstrap';
-import React from 'react';
+import { Button, FormControl } from '@freecodecamp/ui';
+import React, { useCallback, useState } from 'react';
```

Comment:
```suggestion
import React, { useCallback, useState, useRef, useEffect } from 'react';
```

Need these so we can set focus on second dialog.